packages/cypher-builder/src/references/Literal.ts

```diff
@@ -48,4 +50,7 @@ export class Literal<T extends LiteralValue = any> implements CypherCompilable {
     }
 }
 
+/** Represents a NULL literal value
```

Comment:
Are backticks rendered nicely?

```suggestion
/** Represents a `NULL` literal value
```

Reply:
yep